malcolm/core/process.py

```diff
@@ -234,6 +234,25 @@ def add_controller(self, controller, timeout=None):
             if self.state == STARTED and should_publish:
                 self._publish_controllers(timeout)
 
+    def add_controllers(self, controllers, timeout=None):
```

Comment:
Please can you make add_controller(controller) call add_controllers([controller])